client/src/modules/roles/modal/rolesPermissions.html

```diff
@@ -2,7 +2,7 @@
 name="permissionForm"
 bh-submit="RolesPermissionsCtrl.getAffected()"
 novalidate>
-<div class="modal-header">
+<div class="modal-header headercrumb">
```

Comment:
The `headercrumb` should be used with an ordered list.  See [the cash invoice modal](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/modules/cash/modals/invoice-modal.html#L2) for how to do it.

The end result should look something like this:
![headercrumbexample](https://user-images.githubusercontent.com/896472/38094462-b358b19c-3365-11e8-804d-defe32cf82e1.PNG)